openc3/lib/openc3/core_ext/string.rb

```diff
@@ -389,4 +389,20 @@ def to_utf8!
     end
   end
 
-end # class String
+  def comment_erb
+    # Split with -1 to avoid stripping empty strings
+    output = self.split("\n", -1).collect! do |line|
```

Comment:
This could just be self.lines instead of split("\n", -1)

Reply:
I think `self.lines` has different behavior than `split("\n", -1)` when it comes to empty strings